pkg/services/tree/service.go

```diff
@@ -244,6 +304,24 @@ func (s *Service) GetSubTree(req *GetSubTreeRequest, srv TreeService_GetSubTreeS
 		return err
 	}
 
+	var cli TreeService_GetSubTreeClient
+	var outErr error
+	err = s.forEachNode(srv.Context(), cid, func(c TreeServiceClient) bool {
+		cli, outErr = c.GetSubTree(srv.Context(), req)
+		return true
+	})
+	if err != nil {
+		return err
+	} else if outErr != nil {
+		return outErr
+	} else if cli != nil {
```

Comment:
is it possible to get `nil` cli if `outErr` == `nil`?

Reply:
Yes, exactly when the node belongs to a container and the operation is local. Callback is not executed at all in this case.